Core/StitchCoreSDK/Tests/StitchCoreSDKTests/Internal/Net/FoundationHTTPTransportIntTests.swift

```diff
@@ -125,6 +125,7 @@ class FoundationHTTPTransportIntTests: StitchXCTestCase {
             init(_ xcTestCase: XCTestCase) {
                 lineExp = xcTestCase.expectation(description: "lines should be equal")
                 closeExp = xcTestCase.expectation(description: "stream should close")
+                closeExp.assertForOverFulfill = false
```

Comment:
[task] can you add a quick comment explaining why it's okay that we get the state changed event twice?